packages/dnb-eufemia/src/extensions/forms/Field/Indeterminate/Indeterminate.tsx

```diff
@@ -51,7 +55,10 @@ export default function Indeterminate(props: Props) {
     <Toggle
       {...rest}
       variant="checkbox"
-      htmlAttributes={{ indeterminate }}
+      htmlAttributes={{
+        indeterminate,
+        ['aria-controls']: ariaControlsValues,
```

Comment:
```suggestion
        ['aria-controls']: ariaControlsIds,
```

Values vs Ids – I'm not sure. But at the end, they should be Ids when used.

Reply:
renamed `ariaControlsValues` to `ariaControlsIds`.